### ISSUE TITLE
[BUG]: failed transaction

### ISSUE THREAD

**Issue by joticajulian** [koinos/koinos-chain#634]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I tried to submit the transaction in this call

```
    {
      "id": 811,
      "jsonrpc": "2.0",
      "method": "chain.submit_transaction",
      "params": {
        "transaction": {
          "header": {
            "chain_id": "EiDyWt8BeDCTvG3_2QLJWbDJOnHqIcV4Ssqp69aZJsqPpg==",
            "rc_limit": "37188800000000",
            "nonce": "OBs=",
            "operation_merkle_root": "EiCWLn3hLTBvu_fLBnTJQBzJx2bFlY6xi2ZXmNAmqut95g==",
            "payer": "15jmiMAg37PTknTJVzgvMtaYZk5xbtgH5g"
          },
          "operations": [
            {
              "call_contract": {
                "contract_id": "19JntSm8pSNETT9aHTwAUHC5RMoaSmgZPJ",
                "entry_point": 670398154,
                "args": "ChkAM_dMR-sA70UHTD5zv7A02CpHQh5DWSmXEhkAOraorkYwQTkrfp9ViHFI2CJvmCQh2mz7GIDC1y8="
              }
            }
          ],
          "id": "0x12201d44232dc7ba16804797a48da117200e8433e23e89f1bc1a5dde4178b63fe4f7",
          "signatures": [
            "INe4VMuO4inBQxFQLpQrnxKPd1u8gXDwZCOJAYxXEZugDcNNvLDbhpjHJuFcuDWw238xvZ7FUW-JyTgxLmg5YDY="
          ]
        }
      }
    }
```

But it was rejected with the message

```
    {
      code: -32603,
      message: 'could not parse fizzy module',
      data: '{"logs":[]}'
    }
```

any ideas of what is wrong?

### Expected behavior

a transaction submitted

### Environment

```markdown
- OS:
```

**Comment by mvandeberg** [comment]
Some notes from my investigation of the issue:

`19JntSm8pSNETT9aHTwAUHC5RMoaSmgZPJ` is the tKOIN contract. It appears to be working still, so I think we can rule it out as the problem.

`15jmiMAg37PTknTJVzgvMtaYZk5xbtgH5g` is the payer and and contracts were uploaded here. It looks like you are testing the Authorize system. Thank you!

My Contract Meta Store logged the contract being uploaded 6 times.  The last block that it was updated on looks to be 55027, `0x122096fa710daeb80370c4c8c49eea211cfb38ea917b1c85a0edf3f6cd4c841ad531`. Do you know of a recent block where the contract was working?

I see the operation:

```
{
   "upload_contract": {
      "contract_id": "15jmiMAg37PTknTJVzgvMtaYZk5xbtgH5g",
      "bytecode": "V7FycsVI",
      "abi": "test",
      "authorizes_call_contract": true,
      "authorizes_transaction_application": true,
      "authorizes_upload_contract": true
   }
}
```

That looks like an extremely small contract. Could this be a result of user error or is the contract size expected?

**Comment by joticajulian** [comment]
🤦‍♂️ I did exactly what I'm not suppose to do 🤣
I have not tested the authorize system yet. I was updating koilibJS for V3 and I wanted to make sure that the library was building the correct transaction to override the authorize system, pay mana, etc so I created some e2e tests with random bytecodes.
Now I know this is the cause, and the reason I can not use that account anymore.

Fortunately, this is just the testing network. Lesson learned. Thanks for taking a look.
